src/applications/mhv/landing-page/routes.jsx

```diff
@@ -1,10 +1,14 @@
 import React from 'react';
 import { Switch, Route } from 'react-router-dom';
 import App from './containers/App';
+import LandingPage from '../medications/containers/LandingPage';
 
 const routes = (
   <Switch>
-    <Route path="/">
+    <Route exact path="/about-medications" key="medicationLandingPage">
+      <LandingPage />
+    </Route>
+    <Route path="/" key="mhvLandingPage">
```

Comment:
I don't believe it will be necessary to change any code in this file in order to link into the medications app from the `/my-health` landing page.

Reply:
Turns out I was wrong about this assumption. Due to these two components sharing the same base path, it's necessary to route to the MedicationsLandingPage, here.